lib/core/lib/core/fields/event.tsp

```diff
@@ -2,7 +2,7 @@ import "../types.tsp";
 
 namespace CommonGrants.Fields;
 
-using CommonGrants.Types;
```

Comment:
The v1.0 version of the compiler doesn't support referencing absolute namespace paths, so I had to remove any shared part of the namespace. Otherwise the TypeSpec compiler would have raised an error about `CommonGrants.CommonGrants.Types`